src/MiniExcel/Attributes/ExcelColumnAttribute.cs

```diff
@@ -21,6 +21,8 @@ public class ExcelColumnAttribute : Attribute
 
         public bool Ignore { get; set; }
 
+        public bool Formula { get; set; } = false;

```

Comment:
I would suggest making this an Enum with `Value` and `Formula` for now. That way we could easily implement additional types in the future if needed

Reply:
I got you... i added a new enum for support future types.